src/benchmarks/gc/src/suite.py

```diff
@@ -215,22 +215,46 @@ def _normal_benchmarks(proc_count: int) -> Mapping[str, Benchmark]:
     tagb_factor = 0.5 if proc_count == 1 else 1
     return {
         "0gb": Benchmark(
-            arguments=GCPerfSimArgs(tc=proc_count, tagb=300 * tagb_factor, tlgb=0), min_seconds=10
+            arguments=GCPerfSimArgs(
+                tc=proc_count,
+                tagb=300 * tagb_factor,
+                tlgb=0
+            ),
+            min_seconds=10
         ),
         "2gb": Benchmark(
-            arguments=GCPerfSimArgs(tc=proc_count, tagb=300 * tagb_factor, tlgb=2, sohsi=50)
+            arguments=GCPerfSimArgs(
+                tc=proc_count,
+                tagb=300 * tagb_factor,
+                tlgb=2,
+                sohsi=50,
+                pohsi=5,
+                pohpi=100
```

Comment:
I think `-pohpi` could be 0 here. 

`-pohpi` might be interesting when comparing a run using `-sohpi` and equivalent one that uses `-pohpi`. - So that you could have everything as similar as possible except the kind of allocations.

As a setting for before/after test runs, `-pohpi` is unlikely to add a lot value.

Leaving it at 100 is ok as well, bot someone might be confused later about what the purpose.

Reply:
Ok! So for "default" runs '0' would be the best value? I set it to 100 because that's the value GCPerfSim sets as default when the flag is not passed in `ParseFromCommandLine()`.

Reply:
I think I copied the default from soh or poh in GCPerfSim. 0 would be a more sensible default. We should fix that.

Reply:
Ok great! Then I'll include that change as well.

Reply:
Thanks!

Reply:
does this mean there will be allocations on POH? we should not be changing the basic scenarios which have no POH allocations. we can add more that specifically test POH allocations. but none of the current scenarios should change.

Reply:
I had forgot to remove those. Now all POH parameters are set to zero when creating the bench files, which respects the basic scenarios' behavior.

Reply:
are you sure that means there will be no allocations on POH? from a cursory look, I see that `GetPohAllocWeight` seems to set `lohAllocRatio` (can you rename this to `pohAllocRatio`, please?) to 5 if no poh args are specified, ie both `pohAllocInterval` and `lohAllocRatio` are null. @vsadov can you please confirm?

Reply:
`GetPohAllocWeight` argument should be `pohAllocRatio`.  It is a copy-paste error.

Just like with LOH there are more than one ways to specify how many POH objects you may want - in terms of the interval between allocations and in terms of ratio of allocations. 
If both are specified you get an assert.

Reply:
If neither `pohAllocInterval` nor `pohAllocRatio`  are specified, then `pohAllocRatio` is assumed to be 5. 
That is the same behavior as LOH had. I am not sure why that is not 0

Reply:
because there's always LOH allocations; but POH should be completely optional. 

but the main point here is the basic scenarios (0gb, 2gb and etc) that we generate in the suite should not be changed - if you are adding something that causes the default to change, then the args to GCPerfSim in the suite should also change to make sure the basic scenarios don't change.

Reply:
I see. Yes, if we want such behavior for POH, then it seems better if defaults for POH would be all 0. 
I.E.  `DEFAULT_POH_SURV_INTERVAL` and `DEFAULT_POH_INTERVAL` should be changed to 0.